### DISCUSSION TITLE
How do I use a custom 404 page in Next 13?

### DISCUSSION THREAD

**Discussion by SleeplessOne1917** [vercel/next.js#43390] [Help]
I was looking at the docs for Next 13, and so far the only mention of 404 pages I found was the [not-found special file](https://beta.nextjs.org/docs/api-reference/file-conventions/not-found). That not found only gets displayed when the notFound function is called in one of the pages. That's not quite the behavior I'm looking for. I'm looking for a 404 page that will be displayed if a user navigates to a path that doesn't exist, e.g. /nonexistant/path/. Is there a way to do that with Next 13's app directory?

**Comment by ephraimduncan** [comment]
The generic 404 page is not implemented yet. [beta.nextjs.org/docs/app-directory-roadmap](https://beta.nextjs.org/docs/app-directory-roadmap)

Run the notFound() method, it renders not-found.tsx or you can use the `pages` dir
